src/main/scala/shade/memcached/internals/Status.scala

```diff
@@ -10,3 +12,4 @@ case object CASObserveErrorInArgs extends Status
 case object CASObserveModified extends Status
 case object CASObserveTimeout extends Status
 case object IllegalCompleteStatus extends Status
+case class UnhandledStatus(underlyingStatus: OperationStatus) extends Status
```

Comment:
Make this a `final`. Not really necessary, but I feel better when case classes are final because they should be :-)